charts/harvester-cloud-provider/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
   repository: rancher/harvester-cloud-provider
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  tag: "v0.1.1"
```

Comment:
If a tag is expected to be provided here, I think we should omit the default in https://github.com/harvester/charts/blob/153951f5eccc352a812afa74dd4f74b3b0420997/charts/harvester-cloud-provider/templates/deployment.yaml#L22

Reply:
nit: 
```suggestion
  tag: v0.1.1
```